specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/integrationRuntime.json

```diff
@@ -1683,6 +1691,56 @@
         "type": "object"
       }
     },
+    "CopyComputeScaleProperties": {
+      "description": "CopyComputeScale properties for managed integration runtime.",
+      "type": "object",
+      "properties": {
+        "dataIntegrationUnit": {
+          "description": "DIU number setting reserved for copy activity execution. Supported values are multiples of 4 in range 4-256.",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 4
+        },
+        "timeToLive": {
+          "description": "Time to live (in minutes) setting of integration runtime which will execute copy activity.",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 5
+        }
+      },
+      "additionalProperties": {
+        "type": "object"
+      }
+    },
+    "PipelineExternalComputeScaleProperties": {
+      "description": "PipelineExternalComputeScale properties for managed integration runtime.",
+      "type": "object",
+      "properties": {
+        "timeToLive": {
+          "description": "Time to live (in minutes) setting of integration runtime which will execute pipeline and external activity.",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 5
+        },
+        "numberOfPipelineNodes": {
+          "description": "Number of the pipeline nodes, which should be greater than 0 and less than 11.",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 1,
+          "maximum": 10
+        },
+        "numberOfExternalNodes": {
+          "description": "Number of the the external nodes, which should be greater than 0 and less than 11.",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 1,
+          "maximum": 10
+        }
+      },
+      "additionalProperties": {
+        "type": "object"
+      }
```

Comment:
Use of additionalProperties is not allowed for properties owned by the service. The only time its ok to use it is when the properties are pass thru (user defined) and not subject to any validations. Please make this an array or provide an explanation for why you need this.